src/docs/releases/2.1.0.md

```diff
@@ -126,6 +126,12 @@ Additionally, the configuration provider key for the default provider has change
 
 Content item shapes can be overridden by their alias if `AliasPart` is attached or by their slug if `AutoroutePart` is attached. For examples, refer to the [docs](../reference/modules/Templates/README.md).
 
+### Recipes Feature
+
+#### New NamedRecipeStepHandler Base Class
+
+The `NamedRecipeStepHandler` class has been introduced to streamline the implementation of `IRecipeStepHandler`. This base class simplifies the creation of custom recipe steps by handling common step-name check, allowing developers to focus on implementing their specific step logic.
```

Comment:
```suggestion
The `NamedRecipeStepHandler` class has been introduced to streamline the implementation of `IRecipeStepHandler`. This base class simplifies the creation of custom recipe steps by handling common step-name checks, allowing developers to focus on implementing their specific step logic.
```

Reply:
It's a single check not multiple check per step. The context here is a step.

Reply:
That's why it's `checks` not check

Reply:
@hishamco 
 We only do a single check. so it should be `check` not `checks` as per your suggestion.

Reply:
Go it, ignore my comment